source/code/plugins/patch_management_lib.rb

```diff
@@ -405,4 +406,42 @@ def process_update_run(record, tag, host, time)
             return {}
         end
     end
-end
\ No newline at end of file
+	
+    def process_yum_record(record, tag, host, time)
+		@log.debug "LinuxUpdatesProgress: Parsing a record of type #{tag}"
+		update_run = []
+		update_run_record = {}
+		update_run_record["Computer"] = host
+		update_run_record["OSType"] = "Linux"
+		update_run_record["UpdateRunName"] = getUpdateRunName()
+		update_run_record["UpdateTitle"] = record["package-name"]
+		update_run_record["UpdateId"] = SecureRandom.uuid
+		update_run_record["TimeStamp"] = OMS::Common.format_time(time)
+		update_run_record["Tag"] = tag
+
+		update_run_record ["Start-Date"] = DateTime.parse(record["update-action-date"]).strftime(APT_GET_START_DATE_TIME_FORMAT)
```

Comment:
is the update-action-date UTC time? otherwise, you need to convert it to UTC in server side.